docs/user/app_feature_config_to_push.md

```diff
@@ -0,0 +1,99 @@
+# Configuration Postprocessing
+
+!!! note
+    current implementation **only renders the configuration to push, it doesn't update the configuration** into the target devices.
+
+The intended configuration job doesn't produce a final configuration artifact(see below for reasons why). Actually, you should understand it as the "intended" **running** configuration, because the intended configuration job generates what is in the final running configuration. This works well for the "compliance" feature, but not to create a configuration artifact that is ready to push.
```

Comment:
```suggestion
The intended configuration job doesn't produce a final configuration artifact (see below for reasons why). The intended configuration is the "intended" **running** configuration, because the intended configuration job generates what is in the final running configuration. This works well for the "compliance" feature, but not as well to create a configuration artifact that is ready to push.
```